### ISSUE TITLE
Cannot find module '@prisma/client' after installing Recipe

### ISSUE THREAD

**Issue by birksy89** [blitz-js/legacy-framework#535]

**Comment by maietta** [comment]
I wasted nearly 2 days on this problem trying to isolate it.

Turns out, Prisma does NOT work with SvelteKit using NPM in production in some very limited circumstances. (I'm building for SSR and deploying using the Node adaptor to my CapRover instances).

This is the fix:

Ditch NPM for building and instead switch to yarn.

`npm i -g yarn`.

Your commands like `npm run dev` will now become `yarn dev`

Update any workflows to use yarn build, yarn test, etc. Get rid of the --if-present if you use that.

But first, you'll also need to dump the node_modules/ directory as well as your package-lock.json.

Run `yarn install` and you are good to go.

Screw you, npm. I'm done with you!
